js_modules/dagit/packages/core/src/pipelines/SidebarModeSection.tsx

```diff
@@ -119,3 +121,17 @@ const ContextLoggerContainer = styled.div`
     margin-right: 8px;
   }
 `;
+
+const ResourceSourceTag = styled.div`
```

Comment:
We have a `<Tag />` component that you can use instead of using custom styling.  Check out https://dagit-storybook.vercel.app/?path=/story/tag--basic